### PR TITLE
Fix bytes

### PR DESCRIPTION
**Pull request by szepeviktor**
Here is all my knowledge on bytes. I call it byte-level care.
The [Integrity workflow](https://github.com/szepeviktor/byte-level-care/blob/master/.github/workflows/reusable-integrity.yml) checks all these.

1. Remove stray spaces
2. Deduplicate blank lines
3. [Obey EditorConfig](https://github.com/greut/eclint/blob/master/README.md#features) including tabs vs. spaces and line end characters
4. Don't export development files
5. Add LICENSE file

Review without white space changes: https://github.com/Automattic/zoninator/pull/90/files?diff=unified&w=1

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -13,13 +13,20 @@ end_of_line = lf
 insert_final_newline = true
 trim_trailing_whitespace = true
 indent_style = tab
+indent_size = 4
+eclint_block_comment_start = /*
+eclint_block_comment = *
+eclint_block_comment_end = */
 
-[*.yml]
+[{*.yml,*.json}]
```
Comment (GaryJones) [inline .editorconfig:21]: The https://core.trac.wordpress.org/browser/trunk/.editorconfig file doesn't have this, as JSON files for WP-related projects should follow the same 4-space tabs for indentations. 

Can you amend this, and the indentation changes for `composer.json` please?

File: .editorconfig [hunk 2]
```diff
@@ -13,13 +13,20 @@ end_of_line = lf
...
 indent_style = space
 indent_size = 2
 
 [*.md]
 trim_trailing_whitespace = false
 
-[*.txt]
-end_of_line = crlf
+#[*.txt]
+#end_of_line = crlf
```
Comment (GaryJones) [inline .editorconfig:29]: Why is this one commented out?

**Review by GaryJones — CHANGES_REQUESTED** [review]
I'm not usually a fan of so many changes in one PR, but I appreciate the efforts on this. If the JSON indentation issues can be fixed up, this can be merged.

Thanks Viktor!

File: .editorconfig (hunk 1 above)
Comment (szepeviktor) [reply .editorconfig:21]: Done.

File: .editorconfig (hunk 2 above)
Comment (szepeviktor) [reply .editorconfig:29]: Removed.

**Comment by GaryJones** [comment]
I appreciate the quick action! 

Can you change the `composer.json` back to using tab indentation please? Then I'll go ahead and merge this before I touch anything else.

**Comment by szepeviktor** [comment]
All done.
Please do squash my commits.

File: .editorconfig (hunk 1 above)
Comment (GaryJones) [reply .editorconfig]: I see that you've added an entry for `[*.json]`, but this isn't needed - JSON files should inherit the indent_style=tab from `[*]`, and the default indent_size=4, just like the linked editorconfig file in WP core.

Comment (szepeviktor) [reply .editorconfig]: Removed.

File: .editorconfig [hunk 3]
```diff
@@ -21,5 +25,5 @@ indent_size = 2
 [*.md]
 trim_trailing_whitespace = false
 
-[*.txt]
-end_of_line = crlf
+[LICENSE]
+indent_style = unset
```
Comment (GaryJones) [inline .editorconfig:29]:
```suggestion
[*.txt]
end_of_line = crlf

[LICENSE]
indent_style = unset
```

Lets add the `[*.txt]` entry back in, so there's as few deviations from the WP editor config file as possible.

**Review by GaryJones — APPROVED** [review]

File: .editorconfig (hunk 3 above)
Comment (szepeviktor) [reply .editorconfig:32]: That WordPress way resembles typewriters and line printers.

**Comment by szepeviktor** [comment]
Glad to contribute.

**Comment by GaryJones** [comment]
Wonderful - thanks again!

**Pull request merged**